src/org.xtuml.bp.doc/Reference/MASL/MASLConversionGuide/MASLConversionGuide.md

```diff
@@ -131,6 +131,13 @@ When "Export MASL domain" runs, BridgePoint sees the events created with this na
 and generates a valid textual MASL model with concrete polymorphic events specified in the state
 transition tables.  
  
+### MASL projects
+
+There are some situations when the editor must distinguish between a MASL domain
+and MASL project. Because domains are where the majority of the work will take
+place, no special annotation is needed. For MASL projects however, the package
+that contains a project component must contain the string "masl_project".
```

Comment:
This needs to say that the special string goes in the package _description_.  Also, need to generate the html with pandoc and include the html in the pull request.

Reply:
done